examples/GoProControl/GoProControl.ino

```diff
@@ -42,13 +42,37 @@ void loop()
     break;
 
   case 's':
-    Serial.println("Status:");
-    gp.getStatus();
+    
```

Comment:
Since this is only related to hero3 you should remove this (line 45-67) or add it under an if statement which is executed only in case the camera is an hero3

Reply:
ok I will make something working for all camera